index.html

```diff
@@ -1,45 +1,56 @@
 <!doctype html>
-<html lang="en" data-framework="es6">
+<html lang="en" data-framework="javascript">
 	<head>
 		<meta charset="utf-8">
-		<title>Vanilla ES6 • TodoMVC</title>
+		<title>VanillaJS • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
```

Comment:
So I just looked at the source for this package... `todomvc-common` is apparently only for the learn-bar on the main site? We should if we can remove it and probably remove it from the other tutorials as well if it's just a dependency that doesn't actually get used.

Reply:
Good spot! 🔪